petar-stoyanov/homework6/register.php

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Registration form</title>
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+<header>
+	<h1>Registration Form</h1>
+</header>
+<main>
+	<?php
+		$add="";
+		$rem="";
+		$countries = array("Bulgaria");
```

Comment:
@petar-stoyanov7 

Credit for the idea. But the vriable is not persisted between the different submits. After the last lesson(working with files) you should know how to persist it : )